src/main/java/de/sakpaas/backend/service/FavoriteRepository.java

```diff
@@ -0,0 +1,13 @@
+package de.sakpaas.backend.service;
+
+import de.sakpaas.backend.model.Favorite;
+import de.sakpaas.backend.model.Location;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface FavoriteRepository extends JpaRepository<Favorite, Long> {
+  List<Favorite> findByUserUuid(UUID userUuid);
+
+  List<Favorite> findByLocation(Location location);
```

Comment:
Sollte man hier nicht `findByLocationAndUuid` machen? Sonst hat man im Ernstfall in einem Laden in Köln auf einmal 300 Einträge oder so, obwohl einem nur die eine UUID interessiert.

Reply:
FindByLocation wird nur genutzt um Locations zu löschen. :)

Reply:
Ah okay, danke :) Die Idee hat mir David auch gerade versucht zu sagen :D